install/threesdk/cli.py

```diff
@@ -110,15 +110,66 @@ def version():
     Print version number
     """
     print(f"3sdk {_sdk.__version__}")
-    # from InstallTools import get_newest_version
-    dic = _sdk.InstallTools.Tools.get_newest_version()
     # campare with __version__
-    if not dic["up_to_date"]:
+    if not _sdk.InstallTools.Tools.is_latest_release():
+        dic = _sdk.InstallTools.Tools.get_latest_release()
         print(
-            f"Your SDK version is not up-to-date. Newest release is {dic['last_release']}\nPlease visit: {dic['last_release_url']}"
+            f"Your SDK version is not up-to-date. Newest release is {dic['latest_release']}\nPlease visit: {dic['latest_release_url']}\nOr run '3sdk --update'"
         )
 
 
+def update():
+    """
+    """
+    try:
+        from threesdk import container
+        from threesdk.InstallTools import JumpscaleInstaller, MyEnv, Tools
+    except ImportError:
+        from .core import container
+        from .core.InstallTools import JumpscaleInstaller, MyEnv, Tools
+
+    if not Tools.is_latest_release():
+        containers_names = container._containers.IT.DockerFactory.containers()
```

Comment:
there are better ways to access IT, check core module

Reply:
replaced with InstallTools that already imported :+1: